app/build.gradle

```diff
@@ -120,7 +120,6 @@ project.afterEvaluate {
 }
 
 dependencies {
-    implementation libraries.app.calligraphy
```

Comment:
could you also remove it from the dependencies.gradle file?

Reply:
I think I actually already removed it, I don't know why it's not shown

Reply:
It's lost in the noise